o-fish-ios/ViewModel/Report/CatchViewModel.swift

```diff
@@ -26,7 +26,17 @@ class CatchViewModel: ObservableObject, Identifiable {
         case tons = "Tons"
         case tonnes = "Tonnes"
 
-        static let allCases = [pound, kilograms, tons, tonnes]
+        static var allCases: [CatchViewModel.UnitSpecification] {
+            let locale = Locale.current
+            guard let languageCode = locale.languageCode,
+                let regionCode = locale.regionCode else {return [kilograms, tonnes, pound, tons]}
+            if languageCode == "en" && regionCode == "US" {
```

Comment:
this `if` statement should be reformulated and added to guard, so in result you will have only 2 return statements, instead of 2x of `[kilograms, tonnes, pound, tons]`